tests/bgp/test_bgp_bbr.py

```diff
@@ -76,7 +76,8 @@ def add_bbr_config_to_running_config(duthost, status):
             }
         }
     ]
-    json_patch = format_json_patch_for_multiasic(duthost=duthost, json_data=json_patch)
+    # change is applied to asic namespaces only
+    json_patch = format_json_patch_for_multiasic(duthost=duthost, json_data=json_patch, is_asic_specific=True)
```

Comment:
>is_asic_specific [](http://example.com/codeflow?start=88&length=16)

If this is asic specific patch, we should have namespace or scope in patch path.

The following change has same comment.